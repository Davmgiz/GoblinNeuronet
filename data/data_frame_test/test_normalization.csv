test_normalization
0,0,3,4
0,0,5,4
0,0,1,-5
