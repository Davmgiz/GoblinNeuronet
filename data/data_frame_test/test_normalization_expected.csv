test_normalization_expected
0,0,0.6,0.8
0,0,1,0.8
0,0,0.2,-1
